service/callback_helper.go

```diff
@@ -0,0 +1,43 @@
+package service
+
+import (
+	"fmt"
+	"github.com/companieshouse/payments.api.ch.gov.uk/models"
+	"net/http"
+	"os"
+
+	"github.com/companieshouse/chs.go/log"
+)
+
+// redirectUser redirects user to the provided redirect_uri with query params
+func redirectUser(w http.ResponseWriter, r *http.Request, redirectURI string, state string, ref string, status string) {
+	// Redirect the user to the redirect_uri, passing the state, ref and status as query params
+	req, err := http.NewRequest("GET", redirectURI, nil)
+	if err != nil {
+		log.ErrorR(req, fmt.Errorf("error redirecting user: [%s]", err))
+		os.Exit(1)
```

Comment:
should not be doing an os exit here, need to return the error to the client with response code e.g. 
```
w.WriteHeader(http.StatusXXXXX)
return
```